app/src/utils/data/grantRounds.ts

```diff
@@ -87,18 +94,18 @@ export async function getGrantRound(blockNumber: number, grantRoundAddress: stri
       const { provider, multicall } = useWalletStore();
       const data = localStorageData?.data || {};
 
-      // read from registry on first load and every 10 mins
+      // read from registry on first load and every 30 secs after (we need to pick up any changes)
```

Comment:
@gdixon Why do we special case this to run every 30 seconds instead of just running each block, which is every ~13 seconds?